e2e/tools/validator/validator.bm.yaml

```diff
@@ -0,0 +1,29 @@
+log_level: warn # Logging level, defaults is warn
+
+prometheus:
+  job: metal # Job name for baremetal metrics
+  url: http://localhost:9091 # Prometheus server URL
+  rate_interval: 21s  # Rate interval for Promql, default is 20s, typically 4 x $scrape_interval
+  step: 3s  # Step duration for Prometheus range queries
+
+config: # default settings
+  isolated_cpu: "15" # Logical processor that is fully isolated from scheduler (ex. isolcpus)
+  load_curve: "0:15,10:20,25:20,50:20,75:20,100:30,75:20,50:20,25:20,10:20,0:15"
```

Comment:
Why can't we add a new load curve in the existing stressor and not make it part of the config as it overcomplicated the config part How's this different from what [default load curve](https://github.com/sustainable-computing-io/kepler/blob/021b5447285065b7d34b6901da57c6b8c82624bd/e2e/tools/validator/scripts/stressor.sh#L32) gives us?